src/dmstor.cpp

```diff
@@ -20,66 +20,86 @@ vm[] = {
 	.0002908882086657216,
 	.0000048481368110953599
 };
+/* byte sequence for Degree Sign U+00B0 in UTF-8. */
+	static constexpr char
+DEG_SIGN1 = '\xc2';
+	static constexpr char
+DEG_SIGN2 = '\xb0';
+
 	double
 dmstor(const char *is, char **rs) {
 	return dmstor_ctx( pj_get_default_ctx(), is, rs );
 }
 
 	double
 dmstor_ctx(PJ_CONTEXT *ctx, const char *is, char **rs) {
-	int n, nl;
+	int n, nl, adv;
```

Comment:
it would be better to move the declaration of the variable with its initialization at line 68

Reply:
this isn't resolved

Reply:
Agreed, thanks @rouault I've updated my branch today with this change.